spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/outbound/PubSubMessageHandler.java

```diff
@@ -71,8 +73,14 @@
 	private HeaderMapper<Map<String, String>> headerMapper = new PubSubHeaderMapper();
 
 	public PubSubMessageHandler(PubSubOperations pubSubTemplate, String topic) {
+		this(pubSubTemplate, topic, new SerializingConverter());
+	}
+
+	public PubSubMessageHandler(PubSubOperations pubSubTemplate, String topic,
+			Converter<Object, byte[]> payloadConverter) {
 		this.pubSubTemplate = pubSubTemplate;
 		this.topicExpression = new LiteralExpression(topic);
+		this.payloadConverter = payloadConverter;
```

Comment:
I guess we need here `Assert` for all the ctor args. Otherwise we end up with `NPE` at runtime.